tests/unix_stream.rs

```diff
@@ -37,12 +37,14 @@ fn unix_stream_send_and_sync() {
     target_os = "hurd",
     ignore = "getting pathname isn't supported on GNU/Hurd"
 )]
+#[cfg_attr(target_os = "cygwin", ignore)]
```

Comment:
Why are these ignored?

Reply:
Some of them are ignored mistakenly.

I leave two of them ignored. The `accept` method needs `connect` to handshake, but the non-blocking `connect` doesn't handshake. It postpones this process to a `poll`. The ignored tests doesn't poll the client socket, so the spawned thread always blocks.